spec/recipes/runit_spec.rb

```diff
@@ -1,12 +1,12 @@
 require 'spec_helper'
 
 describe 'enterprise::runit' do
-  let(:runner) { ChefSpec::Runner.new }
+  let(:runner) { ChefSpec::SoloRunner.new }
   subject(:chef_run) { runner.converge(described_recipe) }
 
   before :each do
     # ChefSpec told me to do this
-    stub_command("grep 'SV:123456:respawn:/opt/opscode/embedded/bin/runsvdir-start' /etc/inittab")
+    stub_command("grep 'SV:123456:respawn:/embedded/bin/runsvdir-start' /etc/inittab")
```

Comment:
It was because we were setting the install path after this and it was blank. Fixed in 4e049b7.